src/Microsoft.DotNet.Darc/DarcLib/Helpers/DependencyFileManager.cs

```diff
@@ -96,6 +96,9 @@ public DependencyFileManager(
         VersionFiles.DotnetToolsConfigJson
     }.ToImmutableHashSet();
 
+    // In VMR repos, Versions.props doesn't contain any dependency versions maintained by automation, so every change is meaningful
+    public static ImmutableHashSet<string> CodeflowDependencyFiles { get; } = NonCodeflowDependencyFiles.Except([VersionFiles.VersionsProps]).ToImmutableHashSet();
```

Comment:
nit: could also be this:

```suggestion
    public static ImmutableHashSet<string> CodeflowDependencyFiles { get; } = [..NonCodeflowDependencyFiles.Except([VersionFiles.VersionsProps])];
```